src/components/change-requests/new-change-request/new-change-request-page/standard-form-field/standard-form-field.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Row, Col, Form, InputGroup, FormControl } from 'react-bootstrap';
+import { exampleAllWorkPackages } from '../../../../../test-support/test-data/work-packages.stub';
+import { wbsPipe } from '../../../../../shared/pipes';
+import styles from './standard-form-field.module.css';
+
+const StandardFormFields: React.FC = () => {
+  return (
+    <div className={'row'}>
+      <Form className={'px-4'}>
+        <Form.Group controlId="exampleForm.ControlTextarea1">
```

Comment:
replace placeholder controlIds with meaningful ids for each form field. This one could probably be "newCR.what" and other fields would have "what" replaced with the field name